docs/deploy-cluster.md

```diff
@@ -0,0 +1,150 @@
+`local_ip`                          | ""                         | IP address which is used to identify this server, combined with the listen port.
+`mock_server`                       | true                       | Start mock server.
+`daemonize`                         | true                       | Whether to run the process as a daemon.
+`pid_file`                          | "pids/nebula-storaged.pid" | File to hold the process id.
+`meta_server_addrs`                 | ""                         | List of meta server addresses, the format looks like ip1:port1, ip2:port2, ip3:port3.
+`store_type`                        | "nebula"                   | Which type of KVStore to be used by the storage daemon.Options can be \"nebula\", \"hbase\", etc.
+`num_io_threads`                    | 16                         | Number of IO threads
+`storage_http_thread_num`           | 3                          | Number of storage daemon's http thread.
+`num_worker_threads`                | 32                         | Number of workers.
+`engine_type`                       | rocksdb                    | rocksdb, memory...
+`custom_filter_interval_secs`       | 24 * 3600                  | interval to trigger custom compaction.
+`num_workers`                       | 4                          | Number of worker threads.
+`rocksdb_disable_wal`               | false                      | Whether to disable the WAL in rocksdb.
+`rocksdb_db_options`                | ""                         | DBOptions, each option will be given as <option_name>:<option_value> separated by.
+`rocksdb_column_family_options`     | ""                         | ColumnFamilyOptions, each option will be given as <option_name>:<option_value> separated by.
+`rocksdb_block_based_table_options` | ""                         | BlockBasedTableOptions, each option will be given as <option_name>:<option_value> separated by.
+`batch_reserved_bytes`              | 4 * 1024                   | Default reserved bytes for one batch operation
+`block_cache`                       | 4                          | BlockBasedTable:block_cache : MB
+`download_thread_num`               | 3                          | Download thread number.
+`min_vertices_per_bucket`           | 3                          | The min vertices number in one bucket.
+`max_appendlog_batch_size`          | 128                        | The max number of logs in each appendLog request batch.
+`max_outstanding_requests`          | 1024                       | The max number of outstanding appendLog requests.
+`raft_rpc_timeout_ms`               | 500                        | RPC timeout for raft client.
+`accept_log_append_during_pulling`  | false                      | Whether to accept new logs during pulling the snapshot.
+`heartbeat_interval`                | 5                          | Seconds between each heartbeat.
```

Comment:
```heartbeat_interval_secs```? So that consistent with other interval configs

Reply:
Good Point & fixed 👍